src/io/scratch-js/toScratchJS.ts

```diff
@@ -622,6 +624,8 @@ export default function toScratchJS(
               return `(new Date().getMinutes())`;
             case "SECOND":
               return `(new Date().getSeconds())`;
+            default:
+              return `('')`;
```

Comment:
If this `default` doesn't exist, everything will fall through. An empty string matches 2.0 behavior for when this menu is set to an invalid value.